src/main/kotlin/no/nav/familie/ef/mottak/api/SøknadskvitteringController.kt

```diff
@@ -47,4 +48,9 @@ class SøknadskvitteringController(
     fun skolepenger(
         @RequestBody søknad: SøknadMedVedlegg<SøknadSkolepenger>,
     ): Kvittering = okEllerKastException { søknadKvitteringService.mottaSkolepenger(søknad) }
+
+    @PostMapping("arbeidssoker")
+    fun arbeidssøker(
+        @RequestBody skjemaForArbeidssøker: SkjemaForArbeidssøker,
```

Comment:
Det skal ikke være likt som de andre skjemaene da?
```suggestion
        @RequestBody søknad: SøknadMedVedlegg<SøknadArbeidssøker>,
```

Reply:
nei, tror det er en egen type

Reply:
Det er riktig at man ikke trenger å wrappe denne inn i en `SøknadMedVedlegg` 😊